lib/plugins/inspec-sign/lib/inspec-sign/base.rb

```diff
@@ -130,6 +135,14 @@ def read_profile_metadata(path_to_profile)
 
         yaml
       end
+
+      def write_profile_content_id(path_to_profile, profile_content_id)
+        p = Pathname.new(path_to_profile)
+        p = p.join("inspec.yml")
+        yaml = YAML.load_file(p.to_s)
+        yaml["profile_content_id"] = profile_content_id
+        File.open("#{p}", "w") { |f| f.write yaml.to_yaml }
```

Comment:
My only concern here is that we will lose any comments or blank lines in the metadata. Is there any way we could preserve them?